string nombre
uint8 edad
string[] idiomas
